bc_obps/service/data_access_service/document_service.py

```diff
@@ -3,11 +3,11 @@
 
 class DocumentDataAccessService:
     @classmethod
-    def create_statutory_declaration(cls, user_guid, statutory_declaration, operation):
+    def create_statutory_declaration(cls, user_guid, statutory_declaration):
         document = Document.objects.create(
             file=statutory_declaration,
             type=DocumentType.objects.get(name="signed_statutory_declaration"),
             created_by_id=user_guid,
         )
-        operation.documents.set([document])
```

Comment:
I moved this to better separate the data access services

Reply:
For your consideration, perhaps this could be a generic parameterized function?  Something like:
```
   @classmethod
    def create_document_file(cls, user_guid, file_data, document_type_name):
        document_type = DocumentType.objects.get(name=document_type_name)
        document = Document.objects.create(
            file=file_data,
            type=document_type,
            created_by_id=user_guid,
        )
        return document
```